### DISCUSSION TITLE
Trouble sorting GraphQL query on a reference object

### DISCUSSION THREAD

**Discussion by mvonbodun** [gatsbyjs/gatsby#28453] [Help]
I have GraphQL query that pulls data from two referenced objects.  I am trying to sort on a field in the child object (reference) but it never honors the sort:
`query MyQuery {
  allContentfulPartnerCategory(filter: {node_locale: {eq: "en-US"}}, sort: {fields: partner___sequence, order: ASC}) {
    edges {
      node {
        title
        partner {
          name
          shortDescription
          logo {
            file {
              url
            }
          }
          sequence
        }
      }
    }
  }
}`
I've tried the ASC and DESC keywords and they just don't seem to be honored.  I can sort on the parent object.  Any ideas?

Thank you.

**Comment by vladar** [comment]
Hi @mvonbodun !

Sorry to hear you're running into an issue. To help us best begin debugging the underlying cause, it is incredibly helpful if you're able to create a [minimal reproduction][reproduction]. This is a simplified example of the issue that makes it clear and obvious what the issue is and how we can begin to debug it.

If you're up for it, we'd very much appreciate if you could [provide a minimal reproduction][reproduction] and we'll be able to take another look.

Thanks for using Gatsby! 💜

[reproduction]: https://gatsby.dev/reproduction
